docs/docfx.json

```diff
@@ -30,6 +44,16 @@
         ]
       }
     ],
+    "groups": {
```

Comment:
`groups` names should match `moniker_ranges` in `.openpublishing.publish.config.json`